proto/backend.proto

```diff
@@ -81,9 +81,6 @@ message PluginContext {
 
   // The user agent of the Grafana server that initiated the gRPC request.
   string userAgent = 8;
-
-  // The API version that initiated a request
-  string apiVersion = 9;
```

Comment:
Sure this is fine even though not in use? Have you tested it? Like if a new field 9 is added later with a different type and there are grafana/plugin versions still depending on the old version etc.  You never delete anything is what I've learned. I think there's a trick to keep the index unnamed or something